### ISSUE TITLE
Teespring EU

### ISSUE THREAD

**Issue by mrfoto** [jmhobbs/cultofthepartyparrot.com#37]
There's a parrot party in EU as well so why not add EU distribution now that you can
https://teespring.com/teespring-europe
![](http://cultofthepartyparrot.com/parrots/congaparrot.gif)![](http://cultofthepartyparrot.com/parrots/congaparrot.gif)![](http://cultofthepartyparrot.com/parrots/congaparrot.gif)![](http://cultofthepartyparrot.com/parrots/congaparrot.gif)![](http://cultofthepartyparrot.com/parrots/congaparrot.gif)

**Comment by jmhobbs** [comment]
Paging @Shrugs

**Comment by Shrugs** [comment]
@mrfoto I believe I've set this up correctly. Unfortunately, the fabulous mint green available at the US-local fulfillment centers doesn't seem to be available at the EU centers, so I've substituted a nice bright blue. It should be sufficient party material.

https://teespring.com/partyparrot-eu

I've linked this version of the campaign in the original version, so ideally those looking for the EU-specific fulfillment will find it!
